README.md

```diff
@@ -50,13 +50,17 @@ expect(document.querySelector('#title')).to.contain.html('<em>Tea</em>')
 ```
 
 ### `text(text)`
-Assert that the text of the [HTMLElement][] or combined text of the [NodeList][] is equal to or contains the given text, using [`textContent`][textContent]. You may optionally also provide a `trimmed` chaining flag that will trim this text.
+Assert that the text of the [HTMLElement][] or combined text of the [NodeList][] is equal to or contains the given text, using [`textContent`][textContent]. Chaining flags:
+
+`trimmed`  - will trim the text before comparing\
+`rendered` - will use [`innerText`][innerText] when comparing
```

Comment:
If you simply add the node to the body before testing and remove it, it should work. the `dom4` plugin should polyfill phantomjs' lacking API.